net-im/bitlbee/bitlbee-3.6-r1.ebuild

```
@@ -1,9 +1,9 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{7..10} )
```

```suggestion
PYTHON_COMPAT=( python3_{8..10} )
```
`python3_7` is nop for some time, so we can just remove it.